docs/SUMMARY.md

```diff
@@ -0,0 +1,50 @@
+# Table of contents
+
+* [Introduction](README.md)
+
+## Getting Started
+
+* [Overview](getting-started/overview.md)
+* [Substrate Essentials](getting-started/substrate-essentials.md)
+* [CLI](getting-started/cli/README.md)
+  * [subsembly init](getting-started/cli/init.md)
+  * [subsembly compile](getting-started/cli/subsembly-compile.md)
+  * [subsembly spec](getting-started/cli/subsembly-spec.md)
+
+## Development
+
+* [Runtime Development](development/development.md)
+* [Chain Specification](development/chain-specification.md)
+* [Runtime Execution](development/runtime-running.md)
+
+## Modules
+
+* [Core](modules/core/README.md)
+  * [System](modules/core/untitled.md)
+  * [Executive](modules/core/executive.md)
+  * [Crypto](modules/core/crypto.md)
+  * [Log](modules/core/log.md)
+* [Pre-built](modules/pre-built/README.md)
+  * [Aura](modules/pre-built/aura.md)
+  * [Balances](modules/pre-built/balances.md)
+  * [Timestamp](modules/pre-built/timestamp.md)
+  * [TransactionPayment](modules/pre-built/transactionpayment.md)
+
+## Advanced
+
+* [as-scale-codec](advanced/as-scale-codec.md)
+* [subsembly-core](advanced/subsembly-core.md)
+* [Metadata](advanced/untitled/README.md)
+  * [Dispatcher](advanced/untitled/dispatcher.md)
+
+## Guides
+
+* [Create Your First Subsembly Runtime](guides/create-your-first-subsembly-runtime/README.md)
+  * [Set Up](guides/create-your-first-subsembly-runtime/set-up.md)
+  * [Start the Node](guides/create-your-first-subsembly-runtime/start-the-node.md)
+  * [PolkadotJS](guides/create-your-first-subsembly-runtime/polkadotjs.md)
+* [Start Your Network](guides/start-your-network/README.md)
+  * [Launch a Network](guides/start-your-network/launch-a-network.md)
+  * [Generate Authority Keys](guides/start-your-network/generate-authority-keys.md)
+  * [Launch the Third Node](guides/start-your-network/launch-the-third-node.md)
```

Comment:
- I think that it would be better if the first step is named `Prepare the network` and the third step is called `Start the network` since in the third step we are not only starting the third node but starting the block production for alice nad bob as-well.

- Wouldn't it be better if we first generate the 2 keys and after that run the validating nodes?